Only_ForGit_To_TestRun/Questionnaire_Prep/Flanker_Conflict.R

```diff
@@ -21,19 +21,19 @@ Covariates_Subscales = c(
 
 
 # For Handling Routine
-Do_ZScores = 0
-Do_PCA = 1
+Do_ZScores = 0 
+Do_PCA = 0
 
 ################
-# Create Lists for different operations, which are important for specific things
-PCA_Subscales = c("NFC_NeedForCognition", "ATQ_EffortfulControl", "BSCS_SelfControl" , "TIF_Intellect")
 # For this Check with Corinna and ask her to send her analysis script and add condition in Common then referring to TaskName
+# Corinna says that we do not use PCA but CFA-factor scores from the CEI model that is not forked
+CEI.fac.scores = CEI.fac.scores # TODO @Kat needs to be adjusted depending how it is handles in common
```

Comment:
CEI.fac.scores needs to get variable assigned
c("NFC_NeedForCognition", "ATQ_EffortfulControl", "BSCS_SelfControl" , "TIF_Intellect")